modules/gke/bootstrap.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+set -xeuo pipefail
+
+echo "Starting GCP NVMe SSD setup"
+
+# Install required tools
+if command -v apt-get >/dev/null 2>&1; then
+  apt-get update
+  apt-get install -y lvm2
+elif command -v yum >/dev/null 2>&1; then
+  yum install -y lvm2
+else
+  echo "No package manager found. Please install required tools manually."
+  exit 1
+fi
```

Comment:
We should definitely version lock this, or if there aren't dependencies, install from a binary.

Reply:
Just pinned this to a specific version.

Regarding the dependencies:

```
Reading package lists...
+ apt-get install -y lvm2=2.03.11-2.1
Reading package lists...
Building dependency tree...
Reading state information...
The following additional packages will be installed:
  dmeventd dmsetup libaio1 libbsd0 libdevmapper-event1.02.1 libdevmapper1.02.1
  libedit2 libexpat1 liblvm2cmd2.03 libmd0 thin-provisioning-tools
```

Reply:
Sure, but I thought that we did not want to maintain our own image because of security concerns?

Reply:
Just removed the version pin.

When I was originally working on that custom container, I got pushback because of concerns that a lot of vulnerability scanner noise could come from bootstrap containers.

But if we are all fine with this, I am happy to work on that bootstrap Docker image.